File: CS/WindowsApplication1/ValuesConverter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors.Repository;

namespace WindowsApplication1
{
    public class ValuesConverter
    {



        public static string TrackBarRangeToString(TrackBarRange range)
        {
            return string.Format("{0},{1}", range.Minimum, range.Maximum);
        }

        public static TrackBarRange StringToTrackBarRange(string value)
        {
            TrackBarRange result = TrackBarRange.Empty;
            string[] values = value.Split(',');
            if (values.Length == 2)
            {
                int min = Convert.ToInt32(values[0]);
                int max = Convert.ToInt32(values[1]);
                result = new TrackBarRange(min, max);
            }
            return result;
        }


    }
}

File: VB/WindowsApplication1/ValuesConverter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors.Repository

Namespace WindowsApplication1
	Public Class ValuesConverter



		Public Shared Function TrackBarRangeToString(ByVal range As TrackBarRange) As String
			Return String.Format("{0},{1}", range.Minimum, range.Maximum)
		End Function

		Public Shared Function StringToTrackBarRange(ByVal value As String) As TrackBarRange
			Dim result As TrackBarRange = TrackBarRange.Empty
			Dim values() As String = value.Split(","c)
			If values.Length = 2 Then
				Dim min As Integer = Convert.ToInt32(values(0))
				Dim max As Integer = Convert.ToInt32(values(1))
				result = New TrackBarRange(min, max)
			End If
			Return result
		End Function


	End Class
End Namespace
